cloudformation/dynamodb_customer_table.yml

```diff
@@ -18,6 +18,8 @@ Resources:
           AttributeType: S
         - AttributeName: feideOrganizationId
           AttributeType: S
+        - AttributeName: cristinId
```

Comment:
Generic comment: From what I have seen in videos and other material, it seems that the suggestion is that the keys should be very generic. However, I see that under the scope of "each micro-service owns its own data" this might be an overkill here. So I am bit unsure if we are now doing something that we will pay the price later or it is in fact the proper design.